### ISSUE TITLE
Genetic mutations and becoming an android

### ISSUE THREAD

**Issue by exploitativity** [tgstation/tgstation#46192]
Issue reported from  Round ID: 117186 (/tg/Station Bagil [ENGLISH] [US-WEST] [100% FREE LAG])

Reporting client version: 512.1472

I stayed around a delaminating SM without protection for too long and got the Radioactive mutation. Then, I was fed an apple that turned me into an Android. The mutation remained, sadly.
Trying to fix it revealed some problems. Mutadone didn't do anything. Killing and cloning just had me clone in as an android with the same mutation. Someone attempted to use a genetics machine on me, but said he could not manipulate my genes at all. I tried to podperson out, but couldn't draw blood because androids cannot be syringed and I'm not sure if I had blood or not.
Was about to attempt a brain transplant, but didn't get the opportunity.

Basically, I think it's strange that mutations persist in Androids, who don't seem designed to work with genetics, preventing the issue from being fixed, in a way. Sorry if this was rambly or useless.

**Comment by nemvar** [comment]
Happens when swapping into any rad immune species. Doesn't clear mutations.
